utils/cdk-runner/main.go

```diff
@@ -104,11 +104,15 @@ func main() {
 	stackName := os.Getenv("ACORN_EXTERNAL_ID")
 	event := os.Getenv("ACORN_EVENT")
 
+	if os.Getenv("DRY_RUN") != "true" {
+		logrus.Fatal("uh nope")
+	}
```

Comment:
Is this temporary or will this get replaced at some point?

Also use `DryRunEnvKey` here maybe?